tests/test_utilities.py

```diff
@@ -21,7 +21,7 @@ def setUp(self):
         self.signatureObj = Signature(self.partner_id, self.api_key)
         self.cipher = PKCS1_v1_5.new(self.key.exportKey())
         self.__reset_params()
-        self.utilities = Utilities(self.partner_id, self.public_key, 0)
+        self.utilities = Utilities(self.partner_id, self.api_key, 0)
```

Comment:
can you explain this change?

Reply:
Public key is a of type `byte` but Utilities accepts a `str`, this breaks the test when `api_key.encode("utf-8")` is called when generating a signature.